source/license.md

```diff
@@ -1,10 +1,13 @@
 # <i class="fab fa-creative-commons"></i> License
 
-OpenFisca is free software made available under the [AGPL license](https://choosealicense.com/licenses/agpl-3.0/). This means that you are free to use, install and modify it, and that you have to contribute your changes back to the community. This is the only way a contributive digital common can be sustainable, so we are very happy that you take part in this shared effort!
+OpenFisca is a free software made available under the [AGPL license](https://choosealicense.com/licenses/agpl-3.0/). This means that you are free to use, install and modify it with the promise to contribute back your changes to the community. This is the only way a contributive digital common can be made sustainable.
+Any contributions are welcome as long as they abide by our contribution guidelines. We are happy to have you here at the OpenFisca community.
 
-## Computation results
+## OpenFisca is used as
 
-If you provide results based on OpenFisca usage, crediting the project would be very welcome in order to increase its visibility and to increase its contributor base. More people means more maintenance and more upgrades!
+## Computation results
+<!--Can be rephrased to be welcoming and appreciative. -->
+If you provide results based on OpenFisca usage, crediting the project would be very welcome in order to increase its visibility and contributor base. More people means more maintenance and more upgrades!
```

Comment:
I was wondering if we could rephrase the above sentences to make them simpler and more welcoming.

Reply:
Any suggestion?

Reply:
```suggestion
If you provide results based on OpenFisca usage, crediting the project would be very welcome to increase its visibility and contributor base. More people means more maintenance and more upgrades.
```